### DISCUSSION TITLE
Error: copying system image from manifest list: parsing image configuration: Get

### DISCUSSION THREAD

**Discussion by zhengyazhao** [labring/sealos#3387] [Q&A] [answered]
linux：centos7.9
安装步骤：get https://github.com/labring/sealos/releases/download/v4.2.0/sealos_4.2.0_linux_amd64.tar.gz \
   && tar zxvf sealos_4.2.0_linux_amd64.tar.gz sealos && chmod +x sealos && mv sealos /usr/bin

 执行如下命令：
 sealos run labring/kubernetes:v1.25.0 labring/helm:v3.8.2 labring/calico:v3.24.1 --single  
 
 报错：
[root@k8s-master ~]# sealos run labring/kubernetes:v1.25.0 --single
Flag --single has been deprecated, it defaults to running cluster in single mode when there are no master and node
2023-06-19T17:50:53 info Start to create a new cluster: master [172.24.194.200], worker [], registry 172.24.194.200
2023-06-19T17:50:53 info Executing pipeline Check in CreateProcessor.
2023-06-19T17:50:53 info checker:hostname []
2023-06-19T17:50:53 info checker:timeSync []
2023-06-19T17:50:53 info Executing pipeline PreProcess in CreateProcessor.
Resolving "labring/kubernetes" using unqualified-search registries (/etc/containers/registries.conf)
Trying to pull docker.io/labring/kubernetes:v1.25.0...
WARN[0033] Failed, retrying in 2s ... (1/3). Error: copying system image from manifest list: parsing image configuration: Get "https://production.cloudflare.docker.com/registry-v2/docker/registry/v2/blobs/sha256/81/81d78195b93df01c96b701cf2fad2c2dcbbed9b42631839c0ec04cd48949f4e0/data?verify=1687171256-GmREeHGr3du%2BBqqPoeK24uo9V8U%3D": dial tcp 104.18.122.25:443: i/o timeout
WARN[0069] Failed, retrying in 2s ... (2/3). Error: copying system image from manifest list: parsing image configuration: Get "https://production.cloudflare.docker.com/registry-v2/docker/registry/v2/blobs/sha256/81/81d78195b93df01c96b701cf2fad2c2dcbbed9b42631839c0ec04cd48949f4e0/data?verify=1687171292-FqhKTvJYLdhByPqR77IlaKvpIGc%3D": dial tcp 104.18.121.25:443: i/o timeout
WARN[0104] Failed, retrying in 2s ... (3/3). Error: copying system image from manifest list: parsing image configuration: Get "https://production.cloudflare.docker.com/registry-v2/docker/registry/v2/blobs/sha256/81/81d78195b93df01c96b701cf2fad2c2dcbbed9b42631839c0ec04cd48949f4e0/data?verify=1687171327-Wqduq6CeWhJnazkz5zjMJjLBo58%3D": dial tcp 104.18.125.25:443: i/o timeout

请问这个怎么处理，全新的操作系统，什么也没设置过，只执行如上命令

**Comment by bxy4543** [comment, marked as answer]
It seems to be a network problem, unable to pull to the docker hub image.
